app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
 <h1>Users#show</h1>
 <p>Find me in app/views/users/show.html.erb</p>
+
+  <div class="col-md-8">
+    <% if @user.list.any? %>
+      <h3><%= @user.first_name %> <%= @user.last_name %>'s Lists</h3>
+      <ol class="list">
+        <%= render @list %>
+      </ol>
+    <% end %>
+  </div>
+
+  <ul class="nav navbar-nav navbar-right">
```

Comment:
Any reason you're using navbar elements here? I think you're just looking to float elements to the right. If you clarify I can clean up the css.